camera_calibration/nodes/cameracalibrator.py

```diff
@@ -225,7 +228,7 @@ def main():
     rospy.init_node('cameracalibrator')
     node = OpenCVCalibrationNode(boards, options.service_check, sync, calib_flags, fisheye_calib_flags, pattern, options.camera_name,
                                  checkerboard_flags=checkerboard_flags, max_chessboard_speed=options.max_chessboard_speed,
-                                 queue_size=options.queue_size, no_gui=options.no_gui, no_mono_if_stereo=options.no_mono_if_stereo)
+                                 queue_size=options.queue_size, no_gui=options.no_gui, no_mono_if_stereo=options.no_mono_if_stereo, mask = options.mask)
```

Comment:
the line is 153 chars long, a line break would be more sensible

Reply:
also no whitespaces around = here